--- /etc/pam.d/sshd.orig	2017-02-03 11:32:40.931916700 +0800
+++ /etc/pam.d/sshd	2017-02-03 11:34:28.584575056 +0800
@@ -46,6 +46,7 @@ session    required     pam_env.so # [1]
 # /etc/default/locale, so read that as well.
 session    required     pam_env.so user_readenv=1 envfile=/etc/default/locale
 
+session    optional     pam_exec.so /bin/bash -c 'test "${PAM_USER:0:3}" = "rdp" && pkill -u ${PAM_USER} || true'
 # SELinux needs to intervene at login time to ensure that the process starts
 # in the proper default security context.  Only sessions which are intended
 # to run in the user's context should be run after this.

